docs/src/index.ejs

```diff
@@ -26,7 +26,6 @@
   </script>
 </head>
 <body>
-  <script src="https://cdn.jsdelivr.net/npm/@babel/polyfill@7/dist/polyfill.min.js"></script>
```

Comment:
Not sure that we should revert this as it affects only our doc site.

Reply:
yes, agree with the point - but would rather see this being an atomic change: it would be easier to explain this PR as "reverting these two commits", rather than say that "it is partial revert, here are the things that were preserved". Would vote for forfeiting absolute correctness in favour of management simplicity here